tests/unit/components/cylc/common/filter.spec.js

```diff
@@ -0,0 +1,94 @@
+/**
+ * Copyright (C) NIWA & British Crown (Met Office) & Contributors.
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
```

Comment:
If module level docs are a js think then I think a quick plain-English run down at the start would be nice.

Reply:
Test descriptions should be included in the `describe` and `it` blocks.

Reply:
> If module level docs are a JS thing then I think a quick plain-English run down at the start would be nice.

The run-down of this file is simply that it contains the tests for `src/components/cylc/common/filter.js`